heudiconv/bids.py

```diff
@@ -404,10 +404,10 @@ def get_formatted_scans_key_row(dcm_fn):
     """
     dcm_data = dcm.read_file(dcm_fn, stop_before_pixels=True, force=True)
     # we need to store filenames and acquisition times
-    # parse date and time and get it into isoformat
+    # parse date and time of start of run acquisition and get it into isoformat
     try:
         date = dcm_data.ContentDate
-        time = dcm_data.ContentTime
+        time = dcm_data.AcquisitionTime
```

Comment:
is there AcquisitionDate?  very unlikely but still possible that date could be different in some very rare case of scanning around midnight.

Reply:
I _do_ have AcquisitionDate in my dicoms, and the standard has it listed, so I'll switch out ContentDate with AcquisitionDate.

Reply:
Should be good now.